rpcs3/main.cpp

```diff
@@ -877,6 +892,31 @@ int main(int argc, char** argv)
 			}
 		});
 	}
+	else if (parser.isSet(arg_savestate))
```

Comment:
why is this not preferred over the regular boot?

Reply:
Fixed.